lib/matplotlib/axes/_axes.py

```diff
@@ -2640,11 +2644,15 @@ def get_next_color():
             xt = x + labeldistance * radius * math.cos(thetam)
             yt = y + labeldistance * radius * math.sin(thetam)
             label_alignment = xt > 0 and 'left' or 'right'
+            label_rotation = 'horizontal'
+            if labelrotate:
+                label_rotation = thetamd + (0 if xt > 0 else 180)
```

Comment:
Instead of using `thetamd` I think it would be better to just use `np.rad2deg(thetam)`